### ISSUE TITLE
Post sorting according to create date

### ISSUE THREAD

**Issue by schmaker** [friendica/friendica#11764]
### Is the feature request related to a problem? Please describe.

As #11410 made me able to post in the past, there are some stuff that could make it even more usable for profile visiting audience

### Describe the feature you'd like

It would be nice to make profile timeline sorted by "create date" instead of "received date" (or have both options available) .. also, having "Archive" option on profile might be also handy. 

Also, public timeline (<friendica.url>/network) also does show only 2022 even though i made posts "5 years ago"
![image](https://user-images.githubusercontent.com/14954032/180608919-157f8a25-32d3-4d72-9877-2d5cb2864a8f.png)

### Additional context
What would i like to see on profile of person is:
 - Sort timeline by latest activity 
 - Sort timeline by latest creation date
 - Sort timeline by latest received date
 - Archive respecting creation date instead of receiving date (most important imho)

My use case is persistent family photoalbum where i would like to be able to search in using "archive" function and to share it with "grandmas", but i guess someone else might find other use cases as well.

Thank you guys

**Comment by MrPetovan** [comment]
I feel like the profile timeline should be sorted by creation date only, the received date sorting only makes sense for the Network page. The remaining question is about reshares, they should be sorted at the time they were reshared, not initially created by the original author.

**Comment by schmaker** [comment]
It's not like that at the moment. Should i separate it as a bug?

![image](https://user-images.githubusercontent.com/14954032/180610133-d33e68d4-eb84-4b1b-b39e-6904f58906a1.png)
Created 3 years ago, received 1 hour ago, but this post is shown as first.

Reshares should definitely be placed to time of the reshare instead of post create date, that's for sure

**Comment by MrPetovan** [comment]
No, it was more a counter-point to let users choose between several sort orders, which adds complexity and thus development time.

It is working as expected at the moment, so not a bug, but it definitely isn't ideal like you reported.
